### PR TITLE
Software CLI Reference/ Command Updates

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/705

This PR includes all necessary updates to the Software CLI reference guide, as well as command usage throughout the Software docs. All changes based on https://github.com/astronomer/cloud-cli/blob/cli-convergence/CONVERGENCE_CHANGES.md

### CODE REVIEW

File: software/workspace-permissions.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ If a Workspace Admin invites a user to a Workspace that has 0 Airflow Deployment
 To invite a user to a Workspace via the Astronomer CLI, run:
 
 ```bash
-astro workspace user add <email-address> --workspace-id=<workspace-id> --role=<workspace-role>
+astro workspace user add --email <email-address> --workspace-id <workspace-id> --role <workspace-role>
```
Comment (jwitz) [inline software/workspace-permissions.md:47]: In https://github.com/astronomer/cloud-cli/blob/cli-convergence/CONVERGENCE_CHANGES.md we give an example without any `=`'s, but a lot of other commands make use of `=` in flags. Is there a standard on this?

File: software/cli-reference.md [hunk 2]
```diff
@@ -259,10 +262,10 @@ If you do not specify `--desired-airflow-version`, this command will output a li
...
 
-| Flag                        | Value Type | Usage                                                                                                                      |
-| --------------------------- | ---------- | -------------------------------------------------------------------------------------------------------------------------- |
+| Flag                        | Value Type | Usage                                                                                                                    |
+| --------------------------- | ---------- | ------------------------------------------------------------------------------------------------------------------------ |
 | `--deployment-id`           | String     | The ID of the Deployment for which you want to upgrade Airflow. To find your Deployment ID, run `astro deployment list`. |
-| `--desired-airflow-version` | String     | The Airflow version you're upgrading to (e.g. `1.10.14`).                                                                  |
+| `--desired-airflow-version` | String     | The Airflow version you're upgrading to (e.g. `1.10.14`).                                                                |
```
Comment (paolaperaza) [inline software/cli-reference.md:268]:
```suggestion
| `--desired-airflow-version` | String     | The Airflow version you're upgrading to (e.g. `2.2.0`).                                                                |
```

File: software/cli-reference.md [hunk 3]
```diff
@@ -351,14 +354,12 @@ Creates a Deployment-level service account on Astronomer, which you can use to c
...
-| `--role`                     | String     | The User Role for the new service account. Can be `viewer`, `editor`, or `admin`. The default value is `viewer`.                  |
-| `--system-sa`                | Boolean    | Whether this service account is a System service account. Default value is `false`.                                               |
-| `--user-id`                  | String     | The ID for the new service account.                                                                                               |
+| `--deployment-id` (Required) | String     | The Deployment you're creating a service account for.                                                                           |
+| `--label` (Required)         | String     | The name or label for the new service account.                                                                                  |
+| `--role`                     | String     | The User Role for the new service account. Can be `viewer`, `editor`, or `admin`. The default value is `viewer`.                |
```
Comment (paolaperaza) [inline software/cli-reference.md:362]: Did we delete `--user-id`?

File: software/cli-reference.md [hunk 4]
```diff
@@ -412,19 +413,18 @@ Run `astro deployment update <your-deployment-id> [flags]` to update a Deploymen
...
+| `--cloud-role`          | String     | The ARN for the IAM role.                                                                                                                                                                                     |
+| `--dag-deployment-type` | String     | The DAG deploy method for the Deployment. Can be either `image` or `volume`. The default value is `image`.                                                                                                    |
+| `--nfs-location`        | String     | The location for an NFS volume mount, specified as: `<IP>:/<path>`. Must be specified when `--dag-deployment-type=volume`. Input is automatically prepended with `nfs:/` - do not include this in your input. |
+| `label`                 | String     | The label for the Deployment.                                                                                                                                                                                 |
```
Comment (paolaperaza) [inline software/cli-reference.md:421]:
```suggestion
| `label`                 | String     | The label or name for the Deployment.                                                                                                                                                                                 |
```

File: software/cli-reference.md [hunk 5]
```diff
@@ -520,7 +521,13 @@ Refer to the following sections for information on each subcommand.
 
 ## astro dev init
 
-Initializes a new Airflow project in your working directory. The set of files generated by this command are required to run Airflow locally and can be deployed to an Airflow Deployment on Astronomer.
+Initializes a new Airflow project in your working directory. The set of files generated by this command are required to run Airflow locally and can be pushed to a Deployment on Astronomer.
```
Comment (paolaperaza) [inline software/cli-reference.md:524]:
```suggestion
Initializes a new Astro project in your working directory. The set of files generated by this command are required to run Airflow locally and can be pushed to a Deployment on Astronomer.
```

File: software/cli-reference.md [hunk 6]
```diff
@@ -520,7 +521,13 @@ Refer to the following sections for information on each subcommand.
...
+
+:::info
+
+If you want to deploy your project to a Software Deployment, you must either specify the `--use-astronomer-certified` flag when running `astro dev init` or update the image in your project's `Dockerfile` to a supported Astronomer Certified image.
```
Comment (paolaperaza) [inline software/cli-reference.md:528]:
```suggestion
To deploy your project to a Software Deployment, you must either specify the `--use-astronomer-certified` flag when running `astro dev init` or update the image in your project's `Dockerfile` to a supported Astronomer Certified image. For more information, see [Deploy DAGs via Astro CLI](deploy-cli.md).
```

File: software/cli-reference.md [hunk 7]
```diff
@@ -542,10 +549,11 @@ When you run this command, the following skeleton files are generated in your cu
...
+| Flag                         | Value Type                                                                                                                                                      | Usage                                                                                              |
+| ---------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- |
+| `--airflow-version`          | String                                                                                                                                                          | The version of Airflow you want to use. The default value is the latest Airflow version available. |
+| `--use-astronomer-certified` | `` | Create the new project with the latest version of Astronomer Certified. By default, your project is created with the latest version of Astronomer Runtime. |
```
Comment (paolaperaza) [inline software/cli-reference.md:555]:
```suggestion
| `--use-astronomer-certified` | `` | Create the new project with the latest version of Astronomer Certified. By default, your project is created with the latest version of Astronomer Certified. |
```

File: software/cli-reference.md [hunk 8]
```diff
@@ -542,10 +549,11 @@ When you run this command, the following skeleton files are generated in your cu
...
-| `--airflow-version` | String     | The version of Airflow you want to use. The default value is the latest Airflow version available. |
-| `--name`            | String     | The name for the Airflow project.                                                                  |
+| Flag                         | Value Type                                                                                                                                                      | Usage                                                                                              |
+| ---------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------- |
+| `--airflow-version`          | String                                                                                                                                                          | The version of Airflow you want to use. The default value is the latest Airflow version available. |
```
Comment (paolaperaza) [inline software/cli-reference.md:554]:
```suggestion
| `--airflow-version`          | String                                                                                                                                                          | The version of Airflow you want to use. The default value is the latest Airflow version available for Astronomer Certified. |
```

File: software/cli-reference.md [hunk 9]
```diff
@@ -640,6 +648,16 @@ Runs a script that checks whether all files in your local Airflow project are co
...
 
+## astro login/ logout
+
+Authenticates you to Astronomer.
+
+### Usage
+
+Run `astro login <base-domain>` or `astro logout <base-domain>` to log in or out of your Astronomer platform respectively. This is equivalent to using the login screen of the Software UI.
+
+If you have access to more than one Astronomer platform, each will have a unique `<base-domain>`. When switching between platforms, make sure to log out of one `<base domain>` before logging into another.
```
Comment (paolaperaza) [inline software/cli-reference.md:659]:
```suggestion
If you have access to more than one Astronomer installation, each will have a unique `<base-domain>`. When switching between platforms, make sure to log out of one `<base domain>` before logging into another.
```

File: software/cli-reference.md [hunk 10]
```diff
@@ -640,6 +648,16 @@ Runs a script that checks whether all files in your local Airflow project are co
 
 - [Running the Airflow Upgrade Check Package](https://airflow.apache.org/docs/apache-airflow/stable/upgrade-check.html#upgrade-check)
 
+## astro login/ logout
+
+Authenticates you to Astronomer.
```
Comment (paolaperaza) [inline software/cli-reference.md:653]:
```suggestion
Authenticates you into and out of an installation on Astronomer Software.
```

File: software/cli-reference.md [hunk 11]
```diff
@@ -698,13 +716,14 @@ Creates a new Workspace.
 
 ### Usage
 
-`astro workspace create <new-workspace-name> [flags]`
+`astro workspace create --label=<new-workspace-name> [flags]`
```
Comment (paolaperaza) [inline software/cli-reference.md:719]: Is this `--name` or `--label`? The options below list if as `--name` .Assume it's `--label` - let's make sure everything below is consistent

cc: @sunkickr

File: software/manage-airflow-versions.md [hunk 12]
```diff
@@ -56,7 +56,7 @@ Once you select a version, you can expect to see a banner next to **Airflow Vers
 
 ### via the Astronomer CLI
 
-To initialize the Airflow upgrade process via the Astronomer CLI, first make sure you're authenticated by running `$ astro auth login <your-base-domain>`.
+To initialize the Airflow upgrade process via the Astronomer CLI, first make sure you're authenticated by running `$ astro login <your-base-domain>`.
```
Comment (paolaperaza) [inline software/manage-airflow-versions.md:59]:
```suggestion
To initialize the Airflow upgrade process via the Astronomer CLI, first make sure you're authenticated by running `$ astro login <base-domain>`.
```

File: software/workspace-permissions.md (hunk 1 above)
Comment (paolaperaza) [reply software/workspace-permissions.md:47]: ^Good question - @sunkickr ?

**Review by paolaperaza — APPROVED** [review]
@jwitz Largely LGTM but do take a look at ma comments! TY ✅

File: software/cli-reference.md (hunk 3 above)
Comment (jwitz) [reply software/cli-reference.md:362]: yup, per https://github.com/astronomer/cloud-cli/blob/cli-convergence/CONVERGENCE_CHANGES.md#deployment-service-account-commands

File: software/cli-reference.md (hunk 11 above)
Comment (jwitz) [reply software/cli-reference.md:719]: It's label per https://github.com/astronomer/cloud-cli/blob/cli-convergence/CONVERGENCE_CHANGES.md#workspace-commands

**Pull request merged**